win32/src/winapi/kernel32/resource.rs

```diff
@@ -77,18 +81,18 @@ pub fn find_resource<'a>(
     hInstance: HINSTANCE,
     typ: ResourceKey<&Str16>,
     name: ResourceKey<&Str16>,
-) -> Option<&'a [u8]> {
+) -> Option<Range<usize>> {
```

Comment:
This range is a span within the 32-bit memory right?  For consistency I've been using u32s for these, which helps a bit with keeping straight which kind of things we're talking about.  On wasm usize=u32 anyway but on other platforms it's 32-bit vs 64-bit.